js/src/components/contact-information/phone-number-card/phone-number-card.js

```diff
@@ -117,14 +117,27 @@ const PhoneNumberCard = ( {
 } ) => {
 	const { loaded, data } = phoneNumber;
 	const [ isEditing, setEditing ] = useState( initEditing );
+	const onPhoneNumberVerifiedRef = useRef();
+	onPhoneNumberVerifiedRef.current = onPhoneNumberVerified;
+
+	const { isVerified } = data;
 
 	// Handle the initial UI state of `initEditing = null`.
 	// The `isEditing` state is on hold. Determine it after the `phoneNumber` loaded.
```

Comment:
❓  don't understand this line tbh. What does it mean "The `isEditing` state is on hold"?